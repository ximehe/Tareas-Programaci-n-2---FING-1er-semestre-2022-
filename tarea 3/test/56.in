# implementar esPerfecto
esPerfecto
insertarEnAbb (20,20.0)
esPerfecto
insertarEnAbb (30,30.0)
esPerfecto
insertarEnAbb (10,10.0)
esPerfecto
insertarEnAbb (5,5.0)
insertarEnAbb (15,15.0)
esPerfecto
insertarEnAbb (25,25.0)
esPerfecto
insertarEnAbb (35,35.0)
esPerfecto
insertarEnAbb (2,2.0)
insertarEnAbb (7,7.0)
insertarEnAbb (12,12.0)
insertarEnAbb (17,17.0)
insertarEnAbb (22,22.0)
insertarEnAbb (27,27.0)
insertarEnAbb (32,32.0)
esPerfecto
insertarEnAbb (37,37.0)
esPerfecto
Fin