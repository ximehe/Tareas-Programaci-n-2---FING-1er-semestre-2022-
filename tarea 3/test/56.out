1>#  implementar esPerfecto.
2>El abb es perfecto.
3>Se insertó (20,20.00) en el abb.
4>El abb es perfecto.
5>Se insertó (30,30.00) en el abb.
6>El abb NO es perfecto.
7>Se insertó (10,10.00) en el abb.
8>El abb es perfecto.
9>Se insertó (5,5.00) en el abb.
10>Se insertó (15,15.00) en el abb.
11>El abb NO es perfecto.
12>Se insertó (25,25.00) en el abb.
13>El abb NO es perfecto.
14>Se insertó (35,35.00) en el abb.
15>El abb es perfecto.
16>Se insertó (2,2.00) en el abb.
17>Se insertó (7,7.00) en el abb.
18>Se insertó (12,12.00) en el abb.
19>Se insertó (17,17.00) en el abb.
20>Se insertó (22,22.00) en el abb.
21>Se insertó (27,27.00) en el abb.
22>Se insertó (32,32.00) en el abb.
23>El abb NO es perfecto.
24>Se insertó (37,37.00) en el abb.
25>El abb es perfecto.
26>Fin.
